docs/supported-technologies.asciidoc

```diff
@@ -29,6 +29,8 @@ is executed: `domContentLoadedEventEnd` - `domContentLoadedEventStart`.
 `Fire "load" event`::
 Trigged when the browser finishes loading its document and dependent resources: `loadEventEnd` - `loadEventStart`.
 
+`Time to first byte (TTFB)`::
+Duration from the browser making an HTTP request for the initial document to the first byte of the page being received. TTFB is available from the Navigation Timing API as the `reponseStart` timestamp, and captured as transaction mark `transaction.marks.agent.timeToFirstByte` in the ElasticSearch document.
```

Comment:
Should probably mention that it's relative to the `fetchStart` instead of an absolute timestamp.

Reply:
Probably it a wrong place, we should inform this as part of page-load transaction.